nmdc_automation/workflow_automation/models.py

```diff
@@ -63,20 +47,35 @@ def workflow_process_factory(record: Dict[str, Any]) -> Union[DataGeneration, Wo
     wfe = cls(**record)
     return wfe
 
+def _normalize_record(record: Dict[str, Any]) -> Dict[str, Any]:
+    """ Normalize the record by removing the _id field and converting the type field to a string """
+    record.pop("_id", None)
+    # for backwards compatibility strip Activity from the end of the type
+    record["type"] = record["type"].replace("Activity", "")
+    normalized_record = _strip_empty_values(record)
 
-def get_base_workflow_execution_keys() -> List[str]:
-    """ Return the keys that are common to all workflow executions """
-    keys = set()
-    for k, v in WorkflowExecution.__annotations__.items():
-        if k.startswith("_") or k.startswith("class_"):
-            continue
-        keys.add(k)
-    return list(keys)
-
-
-
-
-
+    # type-specific normalization
+    # add type to Mags Analysis mags_list records
```

Comment:
We are missing adding type for [EukEval](https://microbiomedata.github.io/nmdc-schema/EukEval/)

Reply:
see example data https://github.com/microbiomedata/nmdc-schema/blob/main/src/data/valid/MagsAnalysis.yaml

Reply:
https://github.com/microbiomedata/nmdc_automation/pull/262/commits/44866d056173d5906afb2fa7499f0b71390c83b7